package-lock.json

```diff
@@ -1322,9 +1322,7 @@
     "asap": {
```

Comment:
I think you can remove these changes from pakcage-lock.json, since you won't need react or preact.